docs/token_transferring/neonpass_usage.md

```diff
@@ -57,43 +49,48 @@ The following is an example of transferring NEON tokens from Neon EVM to Solana
 
 > For further help, see our [wallet setup walkthrough](/docs/wallet/metamask_setup).
 
-### Step 2: Connect wallets to NeonPass 
+### Step 2: Connect wallets to NeonPass
 
 :::tip
 
 Remember to use a browser to which both your Solana- and EVM-compatible wallets are attached.
 
 :::
 
-<img src={image1} width="450" style={{ display: 'block', margin: '10px auto' }} />
+<img src={image3} width="450" style={{ display: 'block', margin: '10px auto' }} />
+
+2.1 Select the network you want to connect to from the dropdown in the top right corner.
 
+<img src={image1} width="450" style={{ display: 'block', margin: '10px auto' }} />
 
-2.1 Click **Connect Wallet** to connect your Solana-compatible wallet to NeonPass. Follow the login procedure in your wallet's popup window and ensure it's connected to the network you require (e.g. Mainnet). 
+2.2 Click **Connect Wallet** to connect your Solana-compatible wallet to NeonPass. Follow the login procedure in your wallet's popup window and ensure it's connected to the network you require (e.g. Mainnet).
 
 :::tip
 
 Ensure you have enough SOL/NEON to pay the withdrawal approval fee.
 
 :::
 
-2.2 Repeat this step for your EVM-compatible wallet. 
+2.3 Repeat this step for your EVM-compatible wallet.
 
 After successfully connecting your wallets to NeonPass, the **Connect Wallet** text changes to the name of the network (Solana or Neon), and the public key of your accounts is displayed.
 
 ### Step 3: Conduct transfer
 
-3.1 Use the arrow to determine the direction of transfer (from Solana to Neon EVM or vice versa).
+<Tabs>
+  <TabItem value="neontosolana" label="Neon EVM to Solana" default>
+
+3.1 Use the arrow to determine the direction of transfer (from Neon EVM to Solana).
 
 <img src={image2} width="450" style={{ display: 'block', margin: '10px auto' }} />
 
 3.2 Use the drop-down to select the token you want to transfer and enter the amount you want to send.
```

Comment:
@sukanyaparashar It appears that you duplicated data in the tabs -- it seems that you only need to have the direction image tabbed, yes. After that, (unless I guess wrong WRT the comment on 3.4) the flow is identical >> i.e. your divergence has regrouped >> The structure of the page should reflect this.
i.e. Exit your tab at step 1 if that is the last specific instruction that is dependent on direction and simply use the same steps for the rest of the flow.
If you want to arrange a call for me to explain further -- pls grab a time in my diary.

Reply:
@m4sterbunny , the duplication is needed only for the 3.3 and 3.4 transfer steps, but the images are all different and I think we need to keep them separate for clarity. Let me know if the new fix is fine according to you.